Index: app/src/main/java/ru/practicum/android/diploma/ui/jobdetails/JobDetailsFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package ru.practicum.android.diploma.ui.jobdetails\r\n\r\nimport android.os.Bundle\r\nimport android.view.LayoutInflater\r\nimport android.view.View\r\nimport android.view.ViewGroup\r\nimport androidx.fragment.app.Fragment\r\nimport ru.practicum.android.diploma.databinding.FragmentJobDetailsBinding\r\n\r\nclass JobDetailsFragment : Fragment() {\r\n    private var binding: FragmentJobDetailsBinding? = null\r\n\r\n    override fun onCreateView(\r\n        inflater: LayoutInflater,\r\n        container: ViewGroup?,\r\n        savedInstanceState: Bundle?\r\n    ): View? {\r\n        binding = FragmentJobDetailsBinding.inflate(inflater, container, false)\r\n        return binding?.root\r\n    }\r\n\r\n    override fun onDestroyView() {\r\n        super.onDestroyView()\r\n        binding = null\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/ru/practicum/android/diploma/ui/jobdetails/JobDetailsFragment.kt b/app/src/main/java/ru/practicum/android/diploma/ui/jobdetails/JobDetailsFragment.kt
--- a/app/src/main/java/ru/practicum/android/diploma/ui/jobdetails/JobDetailsFragment.kt	
+++ b/app/src/main/java/ru/practicum/android/diploma/ui/jobdetails/JobDetailsFragment.kt	
@@ -19,6 +19,10 @@
         return binding?.root
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+    }
+
     override fun onDestroyView() {
         super.onDestroyView()
         binding = null
